### DISCUSSION TITLE
Markdown

### DISCUSSION THREAD

**Discussion by isisgoncalves** [oscarlojr/oscarlojr#7] [Q&A] [answered]
Onde estuda/estudou markdown?

**Comment by oscarlojr** [comment, marked as answer]
Estudo pela internet, porém existe um gratuito nessa plataforma aqui: [Clique Aqui](https://www.udemy.com/course/aprenda-markdown/?utm_source=adwords&utm_medium=udemyads&utm_campaign=Webindex_Catchall_la.PT_cc.BR&utm_term=_._ag_114148736799_._ad_485704569539_._de_c_._dm__._pl__._ti_dsa-466335283586_._li_1001765_._pd__._&gclid=Cj0KCQjwpompBhDZARIsAFD_Fp-KENbgHYO2BEQueeNKhQ-JSzqiQCvep5oQ-PJ0i-Y1Hf8mqpa8PSIaAmL5EALw_wcB)
